### PR TITLE
Include both isoform_mskimpact and isoform_override

### PR DESCRIPTION
**Pull request by zhx828**

### CODE REVIEW

**Comment by zhx828** [comment]
@JiaoJiao123 could you take a look? Could you also do full tests to see the newly generated gene document will work properly? Thanks!

File: web/yo/app/scripts/factories/VariantFactory.js [hunk 1]
```diff
@@ -392,8 +392,11 @@ angular.module('oncokbApp').factory('OncoTree', ['$http', 'OncoKB', '_', functio
 angular.module('oncokbApp').factory('ApiUtils', ['$http', function($http) {
     'use strict';
 
-    function getIsoforms() {
-        return $http.get('data/isoformMskcc.json');
+    function getIsoforms(type) {
+        if (type && type === 'msk') {
```
Comment (JiaoJiao123) [inline web/yo/app/scripts/factories/VariantFactory.js]: remove check on type exists

File: web/yo/app/scripts/services/mainutils.js [hunk 2]
```diff
@@ -273,10 +277,16 @@ angular.module('oncokbApp')
         function getIsoform(hugoSymbol) {
```
Comment (JiaoJiao123) [inline web/yo/app/scripts/services/mainutils.js]: mainutils.js#Line296
using resolve directly without "return"

**Comment by JiaoJiao123** [comment]
@zhx828 Just did the integrated test together with the history PR, it works fine to create a new gene doc, and review mode on that new doc works properly. History model and queue model also work as designed.

**Pull request merged**